classes/notification/NotificationAPIManager.php

```diff
@@ -64,6 +64,28 @@ public function getUserNotifications(): array
                         $row['action_url'] = $this->system->router->getUrl("mission");
                     }
                     break;
+                case "rank":
+                    $row['action_url'] = $this->system->router->getUrl("profile");
+                case "level":
+                    $row['action_url'] = $this->system->router->getUrl("profile");
+                case "system":
+                    $row['action_url'] = $this->system->router->getUrl("profile");
+                case "warning":
+                    $row['action_url'] = $this->system->router->getUrl("profile");
+                case "report":
+                    $row['action_url'] = $this->system->router->getUrl("profile");
+                case "battle":
+                    $row['action_url'] = $this->system->router->getUrl("profile");
+                case "challenge":
+                    $row['action_url'] = $this->system->router->getUrl("profile");
+                case "team":
+                    $row['action_url'] = $this->system->router->getUrl("profile");
+                case "marriage":
+                    $row['action_url'] = $this->system->router->getUrl("profile");
+                case "student":
+                    $row['action_url'] = $this->system->router->getUrl("profile");
+                case "inbox":
+                    $row['action_url'] = $this->system->router->getUrl("profile");
```

Comment:
Should most of these point to the relevant pages? Team, inbox, sensei, etc

Reply:
Just placeholders, will point to their respective pages.